MergeRequestDemo/ViewController.swift

```diff
@@ -51,8 +51,14 @@ class ViewController: UIViewController {
     // display on screen.
     @IBAction func checkTempButtonTapped(_ sender: Any) {
         
-        let temperature = getTemperature()
-        temperatureLabel.text = String(temperature) + " F"
+        temperatureLabel.alpha = 0.0
+        
```

Comment:
@iosbrain - This animation is too abrupt. First fade out the old temperature, then fade in the new temperature. This experience should be smooth and pleasing for the user.

Reply:
@iosbrain - The new code looks good. I'll resolve conversations, merge your code into master, and finish this merge request.